xbmc/dialogs/GUIDialogSimpleMenu.cpp

```diff
@@ -135,11 +132,14 @@ bool CGUIDialogSimpleMenu::ShowPlaySelection(CFileItem& item, const std::string&
 
     if (item_new->m_bIsFolder == false)
     {
-      if (m_savePath.empty()) // If not set above (choose playlist selected)
-        m_savePath = item.GetDynPath();
+      std::string path{};
```

Comment:
Remove `{}` not needed. Default ctor will be called automatically.

Reply:
Done